cypress/support/index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /**
+       * Custom command to select DOM element by data-cy attribute.
```

Comment:
same

Reply:
Yes, as it's needed for the proper type infer of cypress fn and so on.